src/js/services/conversation-service.js

```diff
@@ -43,12 +43,11 @@ export function sendChain(sendFn) {
 
 export function sendMessage(text) {
     return sendChain(() => {
-        // add an id just to please React
-        // this message will be replaced by the real one on the server response
         const message = {
-            _id: Math.random(),
             role: 'appUser',
-            text
+            text,
+            _tempId: Math.random(),
+            sent: new Date()
```

Comment:
which makes me think that `_id` is confusing because it's not local... idk what the convention should be... nothing? double-underscores?